hare/algorithm.go

```diff
@@ -65,7 +65,7 @@ type ConsensusProcess struct {
 	terminating       bool
 	cfg               config.Config
 	notifySent        bool
-	pending           map[string]*pb.HareMessage
+	pending           map[string]Message
```

Comment:
Since the early messages go through the same path as any other message i.e. re-validated it must be of type Message. Open a bug and we'll fix this path on a later PR.